client/webserver/api.go

```diff
@@ -153,6 +155,7 @@ func (s *WebServer) apiTrade(w http.ResponseWriter, r *http.Request) {
 		OK:    true,
 		Order: ord,
 	}
+	w.Header().Set("Connection", "close")
```

Comment:
Before I increased the http.Server.WriteTimeout to deal with long /api/trade request times, I was seeing what appeared to be the same request retried automatically (unclear if by the frontend or the http.Server itself).  I'm closing the connection used for this request after reply as a precaution though.

Making /api/trade idempotent w.r.t. some form token and/or asynchronous are changes we should consider.

Reply:
I did not see a second POST *in the browser console* so I don't think it was frontend, but I didn't rule it out entirely.  We had a similar issue with the cancel event handler hanging around too long (with the password) and resending the request, so something equally strange could happen again.